docs/webpack-dev-server.config.js

```diff
@@ -18,7 +18,12 @@ const config = {
       //material-ui requires will be searched in src folder, not in node_modules
       'material-ui/lib': path.resolve(__dirname, '../src'),
       'material-ui': path.resolve(__dirname, '../src'),
+      'material-ui-docs': path.resolve(__dirname, 'src/app'),
```

Comment:
Somehow I thought you might say that. :smile: 

I'm going to take another look at moving it out of the examples before we release 0.15.0, so that the examples are 100% copyable, and if I can't figure it out, lets kill it.